testapps/testapp/src/main/res/raw/msal_config_shared_device_mode.json

```diff
@@ -0,0 +1,16 @@
+{
+  "client_id" : "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
+  "authorization_user_agent" : "DEFAULT",
+  "redirect_uri" : "msauth://com.msft.identity.client.sample.local/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
+  "handle_null_taskaffinity": true,
+  "authorization_in_current_task": false,
+  "shared_device_mode_supported": true,
```

Comment:
What does this do? Or I guess what happens when we don't add this or add this and set to `false`? I'm asking because I thought MSAL Test App was already working with Shared Device.